src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs

```diff
@@ -0,0 +1,782 @@
+            }
+
+            public static class Net472
+            {
+                public static ReferenceAssemblies Default { get; }
+                    = new ReferenceAssemblies(
+                        "net472",
+                        new PackageIdentity(
+                            "Microsoft.NETFramework.ReferenceAssemblies.net472",
+                            NuGetVersion.Parse(ReferenceAssembliesPackageVersion)),
+                        @"build\.NETFramework\v4.7.2")
+                    .WithAssemblyIdentityComparer(DesktopAssemblyIdentityComparer.Default)
+                    .AddAssemblies(ImmutableArray.Create("mscorlib", "System", "System.Core", "System.Data", "System.Data.DataSetExtensions", "System.Net.Http", "System.Xml", "System.Xml.Linq"))
+                    .AddLanguageSpecificAssemblies(LanguageNames.CSharp, ImmutableArray.Create("Microsoft.CSharp"))
+                    .AddLanguageSpecificAssemblies(LanguageNames.VisualBasic, ImmutableArray.Create("Microsoft.VisualBasic"));
+
+                public static ReferenceAssemblies WindowsForms { get; }
+                    = Default.AddAssemblies(ImmutableArray.Create("System.Deployment", "System.Drawing", "System.Windows.Forms"));
+
+                public static ReferenceAssemblies Wpf { get; }
+                    = Default.AddAssemblies(ImmutableArray.Create("PresentationCore", "PresentationFramework", "System.Xaml", "WindowsBase"));
+            }
+
+            public static class Net48
+            {
+                public static ReferenceAssemblies Default { get; }
+                    = new ReferenceAssemblies(
+                        "net48",
+                        new PackageIdentity(
+                            "Microsoft.NETFramework.ReferenceAssemblies.net48",
+                            NuGetVersion.Parse(ReferenceAssembliesPackageVersion)),
+                        @"build\.NETFramework\v4.8")
+                    .WithAssemblyIdentityComparer(DesktopAssemblyIdentityComparer.Default)
+                    .AddAssemblies(ImmutableArray.Create("mscorlib", "System", "System.Core", "System.Data", "System.Data.DataSetExtensions", "System.Net.Http", "System.Xml", "System.Xml.Linq"))
+                    .AddLanguageSpecificAssemblies(LanguageNames.CSharp, ImmutableArray.Create("Microsoft.CSharp"))
+                    .AddLanguageSpecificAssemblies(LanguageNames.VisualBasic, ImmutableArray.Create("Microsoft.VisualBasic"));
+
+                public static ReferenceAssemblies WindowsForms { get; }
+                    = Default.AddAssemblies(ImmutableArray.Create("System.Deployment", "System.Drawing", "System.Windows.Forms"));
+
+                public static ReferenceAssemblies Wpf { get; }
+                    = Default.AddAssemblies(ImmutableArray.Create("PresentationCore", "PresentationFramework", "System.Xaml", "WindowsBase"));
+            }
+        }
+
+        public static class NetCore
```

Comment:
what about `3.0` and `3.1`?

Reply:
See https://github.com/dotnet/roslyn-sdk/pull/329#discussion_r341207318